testing/python/language/test_tilelang_language_warp_vote.py

```
@@ -0,0 +1,410 @@
+"""Tests for warp-vote / warp-ballot / block-sync-with-predicate intrinsics.
+
+Covered intrinsics
+------------------
+T.any_sync          – __any_sync  / __any  (HIP)
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)